libchisel/src/verifyimports.rs

```diff
@@ -281,7 +281,9 @@ impl<'a> VerifyImports<'a> {
             _ => Err(()),
         }
     }
+}
 
+impl<'a> VerifyImports<'a> {
```

Comment:
Could wrap this entire implementation into `[cfg(test)]`, though if you do that, please do in a separate commit.

Reply:
will do

Reply:
Done